packages/react/src/components/ProgressIndicator/ProgressIndicator.story.jsx

```diff
@@ -132,6 +132,45 @@ Skeleton.parameters = {
   },
 };
 
+const onlyMainSteps = [
+  {
+    id: 'step1',
+    label: 'First step is very, very long',
+    secondaryLabel: 'Optional label is very, very long',
+    description: 'This is displayed when step icon is hovered',
+  },
+  {
+    id: 'step2',
+    label: 'Second Step',
+    secondaryLabel: 'Optional label',
+  },
+  {
+    id: 'step3',
+    label: 'Third Step',
+    secondaryLabel: 'Optional label',
+    disabled: true,
+  },
+  { id: 'step4', label: 'Fourth Step', invalid: true },
+  { id: 'step5', label: 'Fifth Step', secondaryLabel: 'Optional label is very, very long' },
+];
+
+export const SpaceEqually = () => {
```

Comment:
Same comment as above. Can we just include this prop in an existing story with a knob that can control it?

Reply:
Done. Removed the story and added knob to https://deploy-preview-2622--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-progressindicator--presentation